doc/man3/EVP_PKEY_size.pod

```diff
@@ -0,0 +1,39 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY_size, EVP_PKEY_bits, EVP_PKEY_security_bits
+- EVP_PKEY information functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ int EVP_PKEY_size(const EVP_PKEY *pkey);
+ int EVP_PKEY_bits(const EVP_PKEY *pkey);
+ int EVP_PKEY_security_bits(const EVP_PKEY *pkey);
+
+=head1 DESCRIPTION
+
+EVP_PKEY_size() returns the size of the key in I<pkey> in bytes.
```

Comment:
Errrrrrr...  The name `EVP_PKEY_size()` suggests that it's the size of a key.  Now, it's a well known asymmetric property that the signature will never be bigger than the size of the key, but quite frankly, the previous documentation was plain *odd*, considering that getting the size of the key is not reserved for signing only.

However, if you do read the documentation for `EVP_SignFinal()`, it does explain the signature / `EVP_PKEY_size()` relationship well enough, so that documentation isn't lost either way.

So what you seem to see as an incorrect rewrite, I consider a but fix.  Who wins?  :wink:

Reply:
I think what you have is fine.

Reply:
Maybe this could be reworded in a more suitable manner?

Reply:
I reworded it so that the documentation you suggested for `EVP_PKEY_bits()` also applies to `EVP_PKEY_size()`.  Although...  Meh, I just looked at DSA  :confused: 

Too late at night for this, I'll get back tomorrow.